### DISCUSSION TITLE
How do you paginate() after getWhere()?

### DISCUSSION THREAD

**Discussion by juliantjg** [JoBinsJP/laravel-repository#14] [Q&A] [answered]
Hello @puncoz ,
I'm trying to paginate only a set of rows from the repository, and tried doing the getWhere() and then paginate():
`$filteredUser = $this->userRepository->setTransformer(UserTransformer::class)->getWhere(['status' => 1])->paginate(10);`
But it returned an error saying "collection:paginate does not exist". Is there a way to paginate not the entire repository but only a certain rows?
Thank you!

**Comment by puncoz** [comment, marked as answer]
hi @juliantjg sorry for the delayed response. You can simply call the eloquent's method for query builder.

eg:

```
$this->getBuilder()->where('status', 1)->paginate();
```
